build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/maxdome_post_startup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sourceMetaData=/opt/ml/metadata/resource-metadata.json
```

Comment:
Nonblocking, but If the file is valid json, we can probably use "jq" to get the field? it will be easier to read and write the code and less error prone.

Reply:
I tried using "jq", but the "jq" editor needs to be installed , hence going with traditional grep. We will get back to it after we test E2E.